tests/integrationv2/test_signature_algorithms.py

```diff
@@ -25,6 +25,14 @@
     Signatures.ECDSA_SECP256r1_SHA256,
     Signatures.RSA_PSS_RSAE_SHA256,
     Signatures.RSA_PSS_PSS_SHA256,
+    Signatures.ECDSA_SHA224,
+    Signatures.ECDSA_SHA1,
+]
+
+# RSA_SHA224, ECDSA_SHA224 signature algorithms is not included in security_policy_20210816[].
+sigs = [
+    Signatures.RSA_SHA224,
+    Signatures.ECDSA_SHA224,
 ]
```

Comment:
Since this is specific to the new test, it should probably be renamed something more specific and moved down to where the test is. Or, these signatures could just be inlined in the parameters.